core/controllers/review_tests_test.py

```diff
@@ -127,7 +127,7 @@ def test_no_user_can_access_unpublished_story_review_sessions_page(self):
     def test_get_fails_when_story_doesnt_exist(self):
         self.get_html_response(
             '/learn/staging/topic/review-test/%s'
-            % 'non-existent-story-url-fragment',
```

Comment:
Removed extra part since it must be within the limit in order to check desired behaviour of the test case.